cn_indian_payroll/cn_indian_payroll/patches/sub_category.py

```
@@ -0,0 +1,409 @@
+
+    data=[
+            {
+            
+            
+            "docstatus": 0,
+            "doctype": "Employee Tax Exemption Sub Category",
+            "exemption_category": "Section 10(14)",
+            "is_active": 1,
+            "max_amount": 0.0,
+            "modified": "2024-11-13 18:43:42.148302",
+            "name": "Uniform Allowance"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)